stylesheets/style.css

```diff
@@ -0,0 +1,99 @@
+body {
+    margin: 0;
+    font-family: 'Courier New', Courier, monospace;
+    display: grid;
+    grid-template-columns: 20% auto 20%;
+    grid-template-rows: .5fr 1fr .25fr;
+    color: #3e3e3e;
+    font-weight: 300;
+    background-color: #A3AC9C
+    /* background: url('../images/background.png') */
+}
+
+header {
+    text-align: center;
+    margin-top: 4em;
+    grid-column: 2/3;
+    grid-row: 1/2;
+}
+
+header img {
+    width: 200px;
+    border-radius: 50%;
+}
+
+header nav li {
+    background-color: #f3f3f3;
+    padding: .5em;
+    border-radius: 10px;
+}
+
+ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+li {
+    list-style-type: none;
+}
+
+a {
```

Comment:
to make it more user friendly I would add an effect to your links so folks know its a link. Maybe when you hover the mouse could change to a pointer or the text could change colors.